osu.Game.Tests/Visual/UserInterface/TestSceneDeleteLocalScore.cs

```diff
@@ -92,6 +89,12 @@ protected override IReadOnlyDependencyContainer CreateChildDependencies(IReadOnl
             dependencies.Cache(scoreManager = new ScoreManager(dependencies.Get<RulesetStore>(), () => beatmapManager, LocalStorage, Realm, Scheduler));
             Dependencies.Cache(Realm);
 
+            return dependencies;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load() => Schedule(() =>
```

Comment:
A bit weird to schedule from load, but seems to work and match the previous intentions.